.organizationmap

```diff
@@ -135,7 +135,10 @@ EISOO <contact@eisoo.com> lu.shasha <lu.shasha@eisoo.com>
 Enproduktion GmbH <contact@produktion.io> Malte Fiala <maltefiala@produktion.io>
 EPAM <contact@epam.com> Andrey Kuznetsov <Andrey_Kuznetsov@epam.com>
 Exalead <contact@exalead.com> <roche+git@exalead.com>
+Easystack <contact@easystack.cn> chenliuzhong <liuzhong.chen@easystack.cn>
```

Comment:
It looks like this is a github username `chenliuzhong`. Please provide your actual name here, which is usually a display name in the Email. It should be in the alphabetical order with in the group `Easystack` too.